src/lib/mongoBackend/TriggeredSubscription.cpp

```diff
@@ -97,9 +99,14 @@ TriggeredSubscription::~TriggeredSubscription()
 * TriggeredSubscription class is shared for NGSI9 and NGSI10 subscriptions, so it is better
 * to keep expressions (an artifact for NGSI10) out of the constructor, in its independent fill
 * method
-*
 */
-void TriggeredSubscription::fillExpression (const std::string& q, const std::string& georel, const std::string& geometry, const std::string& coords)
+void TriggeredSubscription::fillExpression
+(
+  const std::string&  q,
+  const std::string&  georel,
+  const std::string&  geometry,
+  const std::string&  coords
+)
 {
```

Comment:
In the case of noCache, I understand tha the StringFilter has to be build from the DB content. In the DB, the q filter string is stored in some place I don't remember right know :) but checking DB model documentation it should be easy to locate.